Cryptic error in case shared library missing

When running tests `mod_wolfssl` and `mod_sqlite3`, I got these somewhat cryptic messages:
```
~/fuzion/fuzion/build/tests/mod_wolfssl 12:45:03 > make jvm
FUZION_JVM_BACKEND_OPTIONS="-JLibraries=wolfssl"  FUZION_C_BACKEND_OPTIONS="-CLink=wolfssl|-CInclude=wolfssl/options.h wolfssl/ssl.h|-CFlags=-DWC_NO_HARDEN"  ../check_simple_example_jvm.sh "../../bin/fz -modules=wolfssl" mod_wolfssl.fz || exit 1
RUN mod_wolfssl.fz 0a1,18
> 
> error 1: java.lang.ExceptionInInitializerError
> 	at fzC_fuzion_runtime_init.fzRoutine({base.fum}/fuzion_runtime_init.fz)
> 	at fzC_mod_wolfssl.fzRoutine(--CURDIR--/mod_wolfssl.fz:1)
> 	at fzC_universe.fz_run(--builtin--)
> 	at dev.flang.be.jvm.runtime.FuzionThread.lambda$new$1(FuzionThread.java:120)
> 	at dev.flang.util.Errors.runAndExit(Errors.java:912)
> 	at dev.flang.be.jvm.runtime.FuzionThread.lambda$new$2(FuzionThread.java:132)
> 	at java.base/java.lang.Thread.run(Thread.java:1583)
> Caused by: java.lang.IllegalArgumentException: Cannot open library: libwolfssl.so
> 	at java.base/java.lang.foreign.SymbolLookup.libraryLookup(SymbolLookup.java:277)
> 	at java.base/java.lang.foreign.SymbolLookup.libraryLookup(SymbolLookup.java:238)
> 	at dev.flang.be.jvm.runtime.Runtime.get_method_handle(Runtime.java:1405)
> 	at fzC_fzE_init.<clinit>({base.fum}/native.fz)
> 	... 7 more
> 
> *** fatal errors encountered, stopping.
> one error.
*** FAILED err on mod_sqlite.fz
make: *** [../simple.mk:68: jvm] Fehler 1
```
and
```
~/fuzion/fuzion/build/tests/mod_sqlite 12:47:53 > make jvm
FUZION_JVM_BACKEND_OPTIONS="-JLibraries=sqlite3"  FUZION_C_BACKEND_OPTIONS="-CLink=sqlite3|-CInclude=sqlite3.h"  ../check_simple_example_jvm.sh "../../bin/fz " mod_sqlite.fz || exit 1
RUN mod_sqlite.fz 0a1,18
> 
> error 1: java.lang.ExceptionInInitializerError
> 	at fzC_fuzion_runtime_init.fzRoutine({base.fum}/fuzion_runtime_init.fz)
> 	at fzC_universe.fzRoutine(--builtin--:1)
> 	at fzC_universe.fz_run(--builtin--)
> 	at dev.flang.be.jvm.runtime.FuzionThread.lambda$new$1(FuzionThread.java:120)
> 	at dev.flang.util.Errors.runAndExit(Errors.java:912)
> 	at dev.flang.be.jvm.runtime.FuzionThread.lambda$new$2(FuzionThread.java:132)
> 	at java.base/java.lang.Thread.run(Thread.java:1583)
> Caused by: java.lang.IllegalArgumentException: Cannot open library: libsqlite3.so
> 	at java.base/java.lang.foreign.SymbolLookup.libraryLookup(SymbolLookup.java:277)
> 	at java.base/java.lang.foreign.SymbolLookup.libraryLookup(SymbolLookup.java:238)
> 	at dev.flang.be.jvm.runtime.Runtime.get_method_handle(Runtime.java:1405)
> 	at fzC_fzE_init.<clinit>({base.fum}/native.fz)
> 	... 7 more
> 
> *** fatal errors encountered, stopping.
> one error.
*** FAILED err on mod_sqlite.fz
make: *** [../simple.mk:68: jvm] Fehler 1
```
I could solve them using 
```
> sudo apt install libwolfssl-dev
> sudo apt install libsqlite3
```
but took me some research. Would be better to create a simpler error message and explain what needs to be installed and how, at least in a readme. The source of `modules/wolfssl/` also lacks some documentation.

--- a/src/dev/flang/be/jvm/runtime/Runtime.java
+++ b/src/dev/flang/be/jvm/runtime/Runtime.java
@@ -1402,7 +1402,16 @@ public class Runtime extends ANY
       }
     for (String library : libraries)
       {
-        llu = llu.or(SymbolLookup.libraryLookup(System.mapLibraryName(library), arena));
+        try
+          {
+            llu = llu.or(SymbolLookup.libraryLookup(System.mapLibraryName(library), arena));
+          }
+        catch (IllegalArgumentException e)
+          {
+            Errors.error("'" + library + "' not found on your system. "
+                        + "Make sure to install the corresponding package, that provides '" + library + "'.");
+            System.exit(1);
+          }
       }
 
     var memSeg = llu

--- a/src/dev/flang/be/jvm/runtime/Runtime.java
+++ b/src/dev/flang/be/jvm/runtime/Runtime.java
@@ -1402,14 +1402,15 @@ public class Runtime extends ANY
       }
     for (String library : libraries)
       {
+        var ln = System.mapLibraryName(library);
         try
           {
-            llu = llu.or(SymbolLookup.libraryLookup(System.mapLibraryName(library), arena));
+            llu = llu.or(SymbolLookup.libraryLookup(ln, arena));
           }
         catch (IllegalArgumentException e)
           {
-            Errors.error("'" + library + "' not found on your system. "
-                        + "Make sure to install the corresponding package, that provides '" + library + "'.");
+            Errors.error("'" + ln + "' not found on your system. "
+                        + "Make sure to install the corresponding package, that provides '" + ln + "'.");
             System.exit(1);
           }
       }

